app/api/tests.py

```diff
@@ -1,3 +1,15 @@
-# from django.test import TestCase
+from django.test import Client, TestCase
 
-# Create your tests here.
+
+class ApiTest(TestCase):
+    """Test for API"""
+
+    def setUp(self):
+        self.client = Client()
+        self.endpoint = '/api'
+
+    def test_api_endpoint_is_live(self):
+        r = self.client.get(self.endpoint, {'query': ''})
```

Comment:
It's probably a bad sign if you have to resort to bad variable names to keep the line length short.
You can break the arguments of the `assertEqual` function over multiple lines and still have valid code that follows PEP8